pkg/cmd/pulumi-language-java/main.go

```diff
@@ -195,7 +195,7 @@ func (host *javaLanguageHost) determinePulumiPackages(
 	// Run our classpath introspection from the SDK and parse the resulting JSON
 	cmd := exec.Cmd
 	args := exec.PluginArgs
-	output, err := host.runJavaCommand(ctx, exec.Dir, cmd, args)
+	output, err := host.runJavaCommand(ctx, exec.Dir, cmd, args, true)
```

Comment:
LGTM in principle but have a nit:

Ok this is reasonable but note this:

- this is already printing the error into logging.V(3) on L201 so I think perhaps this should also send stderr/stdout via `logging.V(3)` in the case of error

- yes to runJavaCommand not printing stderr/stdout in the case of plugins; we want that to be much more quiet

- but printing to stdout if logging.V(1) is set is very weird - why not just send the data via the logging framework instead of printing to stdout